tomviz/CentralWidget.ui

```diff
@@ -96,15 +68,15 @@
  </widget>
  <customwidgets>
   <customwidget>
-   <class>pqTabbedMultiViewWidget</class>
+   <class>QVTKWidget</class>
    <extends>QWidget</extends>
-   <header>pqTabbedMultiViewWidget.h</header>
+   <header location="global">qvtkwidget.h</header>
```

Comment:
Should be 'QVTKWidget.h'.  Case matters.  OSX's filesystem is a bit more forgiving about this sort of thing.  It was a build error on my Ubuntu machine.